### PR TITLE
pxb-1631: resolved based on the ticket comments

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/throttling_backups.rst [hunk 1]
```diff
@@ -3,14 +3,17 @@
 Throttling Backups
 ==================
 
-Although xtrabackup does not block your database's operation, any backup can add load to the system being backed up. On systems that do not have much spare I/O capacity, it might be helpful to throttle the rate at which xtrabackup reads and writes data. You can do this with the :option:`--throttle` option, this option limits the number of I/O operations per second in 1 MB units. 
+Although xtrabackup does not block your database's operation, any backup can add
+load to the system being backed up. On systems that do not have much spare I/O
+capacity, it might be helpful to throttle the rate at which xtrabackup reads and
+writes data. You can do this with the :option:`--throttle` option.
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/throttling_backups.rst:9]: This page does not mention the chunk size anymore. Is it intentional change? We could mention here that chunk size is 10Mb.

**Review by gl-sergei — CHANGES_REQUESTED** [review]
Please fix typo in the commit message and PR title xpb -> PXB.

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/throttling_backups.rst (hunk 1 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/throttling_backups.rst:9]: The chunk size is mentioned in the description of the --throttle option.

Comment (gl-sergei) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/throttling_backups.rst:9]: Ok. Maybe it is enough.

**Review by gl-sergei — APPROVED** [review]

**Pull request merged**